tests/test_repocard.py

```diff
@@ -136,6 +136,43 @@
 ---
 """
 
+DUMMY_MODELCARD_EVAL_RESULT_BOTH_VERIFIED_AND_UNVERIFIED = """---
+model-index:
+- name: RoBERTa fine-tuned on ReactionGIF
+  results:
+  - task:
+      type: text-classification
+      name: Text Classification
+    dataset:
+      name: ReactionGIF
+      type: julien-c/reactiongif
+      config: default
+      split: test
+    metrics:
+    - type: accuracy
+      value: 0.2662102282047272
+      name: Accuracy
+      config: default
+      verified: false
+  - task:
+      type: text-classification
+      name: Text Classification
+    dataset:
+      name: ReactionGIF
+      type: julien-c/reactiongif
+      config: default
+      split: test
+    metrics:
+    - type: accuracy
+      value: 0.6666666666666666
+      name: Accuracy
+      config: default
+      verified: true
+---
+
+This is a test model card.
```

Comment:
Small suggestion to help community developers understand what this is about:
```suggestion
This is a test model card containing one self-reported metric and one "verified" metric in the format produced by Hugging Face's [model evaluation service](https://huggingface.co/spaces/autoevaluate/model-evaluator).
```